internal/stack/errors.go

```diff
@@ -8,5 +8,6 @@ import "fmt"
 
 // UndefinedEnvError formats an error reported for undefined variable.
 func UndefinedEnvError(envName string) error {
-	return fmt.Errorf("undefined environment variable: %s", envName)
+	return fmt.Errorf("undefined environment variable: %s. If you have started the Elastic stack using the elastic-package tool, "+
```

Comment:
```suggestion
	return fmt.Errorf("undefined environment variable: %s. If you started the Elastic stack using the elastic-package tool, "+
```

Reply:
I'm good with this solution for now but I'm wondering how difficult it would be to detect programmatically if the stack was started up using `elastic-package`. Again, okay to implement something "smarter" like that in a follow up PR if it's not trivial. The changes you have here already are an improvement so I'm ++ to going forward with them for now.

Reply:
I know what you're suggesting, but I think I would stick to the if-sentence, to present what is expected in both cases.